pkg/storage/cacher_test.go

```diff
@@ -87,7 +87,10 @@ func updatePod(t *testing.T, s storage.Interface, obj, old *api.Pod) *api.Pod {
 }
 
 func TestList(t *testing.T) {
+	tm := time.Now()
```

Comment:
If we're going to be doing this often, can we add a stopwatch type thing? IMO it really clutters up the test. Something like this:

```go
func NewStopwatch() *Stopwatch {
  return &Stopwatch{time.Now()}
}

type Stopwatch struct {
  current time.Time
}

// Get the duration since the stopwatch was constructed, or since the previous call to set.
func (s *Stopwatch) Set() time.Duration {
  d := time.Since(s.current)
  s.current = time.Now()
  return d
}
```